src/netbsd.rs

```diff
@@ -25,27 +41,33 @@ fn kern_arnd(buf: &mut [MaybeUninit<u8>]) -> libc::ssize_t {
 type GetRandomFn = unsafe extern "C" fn(*mut u8, libc::size_t, libc::c_uint) -> libc::ssize_t;
 
 // getrandom(2) was introduced in NetBSD 10.0
-static GETRANDOM: LazyPtr = LazyPtr::new();
+static GETRANDOM: AtomicPtr<c_void> = AtomicPtr::new(ptr::null_mut());
```

Comment:
The `getrandom(2)` comment looks out of place here. It's probably better to explain what we do here in the module-level docs.

Reply:
I don't disagree with that, but moving the comment is out of scope for this PR, isn't it? This PR doesn't change anything related to it.

Reply:
Previously this static was used only for `getrandom` pointers, now it also used for the polyfill, so in a certain sense this PR made the comment outdated.

I think it's fine to improve docs in this PR or you can simply remove the comment.

Reply:
OK, I moved and expanded the comment.